src/binaryen-c.cpp

```diff
@@ -1882,28 +1882,27 @@ void BinaryenRemoveExport(BinaryenModuleRef module, const char* externalName) {
 
 // Function table. One per module
 
-void BinaryenSetFunctionTable(BinaryenModuleRef module, BinaryenFunctionRef* funcs, BinaryenIndex numFuncs) {
+void BinaryenSetFunctionTable(BinaryenModuleRef module, const char **funcNames, BinaryenIndex numFuncNames) {
   if (tracing) {
     std::cout << "  {\n";
-    std::cout << "    BinaryenFunctionRef funcs[] = { ";
-    for (BinaryenIndex i = 0; i < numFuncs; i++) {
+    std::cout << "    const char* funcNames[] = { ";
```

Comment:
btw `char* funcNames` vs ``char *funcNames` pointer placement isn't consistent in the codebase but I tried to be consistent with other similar contexts. Happy to follow any pattern.

Reply:
Yeah, we should be more consistent. I think we prefer `char* funcNames` (keep the type all together).